algorithms/ollie/api/swagger/swagger.yaml

```diff
@@ -27,7 +27,7 @@ paths:
         \ in the text by running ollie algorithm on it. The response includes the\
         \ sentence and the instances which contains detail about each instance of\
         \ relationship found. \n"
-      operationId: "getRelationshipsGET"
+      operationId: "getRelationshipsPOST"
```

Comment:
yeah True, but actually swagger automatically add the GET and POST to the operations. But for our purpose I have renamed them